src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/CountOperation.java

```diff
@@ -0,0 +1,53 @@
+package io.stargate.sgv2.jsonapi.service.operation.model;
+
+import io.smallrye.mutiny.Uni;
+import io.stargate.bridge.proto.QueryOuterClass;
+import io.stargate.sgv2.api.common.cql.builder.BuiltCondition;
+import io.stargate.sgv2.api.common.cql.builder.QueryBuilder;
+import io.stargate.sgv2.jsonapi.api.model.command.CommandContext;
+import io.stargate.sgv2.jsonapi.api.model.command.CommandResult;
+import io.stargate.sgv2.jsonapi.exception.ErrorCode;
+import io.stargate.sgv2.jsonapi.exception.JsonApiException;
+import io.stargate.sgv2.jsonapi.service.bridge.executor.QueryExecutor;
+import io.stargate.sgv2.jsonapi.service.operation.model.impl.CountOperationPage;
+import io.stargate.sgv2.jsonapi.service.operation.model.impl.DBFilterBase;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Supplier;
+
+/**
+ * Full dynamic query generation for any of the types of filtering we can do against the the db
```

Comment:
Is this Javadoc accurate? Seems like it describes something more generic (or omits mention it relates to "count" queries?)

Reply:
Fixed it